_ml-commons-plugin/algorithms.md

```diff
@@ -412,23 +423,27 @@ The Localization algorithm can only be executed directly. Therefore, it cannot b
 
 A classification algorithm, logistic regression models the probability of a discrete outcome given an input variable. In ML Commons, these classifications include both binary and multi-class. The most common is the binary classification, which takes two values, such as "true/false" or "yes/no", and predicts the outcome based on the values specified. Alternatively, a multi-class output can categorize different inputs based on type. This makes logistic regression most useful for situations where you are trying to determine how your inputs fit best into a specified category. 
 
+**Optimisers supported:** `SIMPLE_SGD`, `LINEAR_DECAY_SGD`, `SQRT_DECAY_SGD`, `ADA_GRAD`, `ADA_DELTA`, `ADAM`, and `RMS_PROP`.  
+**Objectives supported:** `HINGE`, and `LOGMULTICLASS`.  
+**momentum_type supported:** `STANDARD`, and `NESTEROV`.  
+
 ### Parameters
 
 | Parameter | Type | Description | Default value |
 |---|---|---|---|
-| `learningRate` | Double | The initial step size used in an iterative optimization algorithm. | `1` |
-| `momentumFactor` | Double | The extra weight factors that accelerate the rate at which the weight is adjusted. This helps move the minimization routine out of local minima. | `0` |
+| `learning_rate` | Double | The initial step size used in an iterative optimization algorithm. | `1` |
+| `momentum_factor` | Double | The extra weight factors that accelerate the rate at which the weight is adjusted. This helps move the minimization routine out of local minima. | `0` |
 | `epsilon` | Double | The value for stabilizing gradient inversion. | `0.1` |
 | `beta1` | Double | The exponential decay rates for the moment estimates. | `0.9` |
 | `beta2` | Double | The exponential decay rates for the moment estimates. | `0.99` |
-| `decayRate` | Double | The Root Mean Squared Propagation (RMSProp). | `0.9` |
-| `momentumType` | String | The Stochastic Gradient Descent (SGD) momentum that helps accelerate gradient vectors in the right direction, leading to faster convergence between vectors. | `STANDARD` |
-| `optimizerType` | String | The optimizer used in the model.  | `AdaGrad` |
+| `decay_rate` | Double | The Root Mean Squared Propagation (RMSProp). | `0.9` |
+| `momentum_type` | String | The Stochastic Gradient Descent (SGD) momentum that helps accelerate gradient vectors in the right direction, leading to faster convergence between vectors. | `STANDARD` |
+| `optimiser` | String | The optimizer used in the model.  | `ADA_GRAD` |
```

Comment:
Is this the correct parameter name spelling? The American spelling is "optimizer".